src/EnergyPlus/SolarShading.cc

```diff
@@ -3679,25 +3685,58 @@ namespace SolarShading {
 
 		//   Note -- if not the below, values are set in SkyDifSolarShading routine (constant for simulation)
 		if ( DetailedSkyDiffuseAlgorithm && ShadingTransmittanceVaries && SolarDistribution != MinimalShadowing ) {
-			CosPhi = 1.0 - SUNCOS( 3 );
+			WithShdgIsoSky = 0.;
```

Comment:
The decimal point after the zero here is completely fine, but also unnecessary.  The compiler knows it is a floating point, so even just `0` is fine.